src/vs/workbench/browser/parts/editor/breadcrumbs.ts

```diff
@@ -142,6 +143,17 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 				localize('symbolpath.last', "Only show the current symbol in the breadcrumbs view."),
 			]
 		},
+		'breadcrumbs.outlineSort': {
```

Comment:
I would call the setting `breadcrumbs.symbolSortOrder` because we already use `symbol` in `breadcrumbs.symbolPath` and `sortOrder` is more explicit.

Reply:
Good one. Changed.